packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -44,86 +68,57 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
   }
 
   public render() {
-    let stars: JSX.Element[] = [];
-    for (let i = this.props.min as number; i <= (this.props.max as number); ++i) {
-      stars.push(this._renderStar(i));
+
+    let stars = [];
+
+    for (let i = this.props.min as number; i <= (this.props.max as number); i++) {
+      let ratingStarProps: IRatingStarProps = {
+        fillPercentage: this._getFillingPercentage(i),
+        disabled: this.props.disabled ? true : false
+      };
+
+      stars.push(
+        <button
+          className={ css('ms-Rating-button', styles.ratingButton, {
+            ['ms-Rating--large ' + styles.rootIsLarge]: this.props.size === RatingSize.Large,
+            ['ms-Rating--small ' + styles.rootIsSmall]: this.props.size !== RatingSize.Small
```

Comment:
Is that correct? If the size is large, we add a "large" style, if it's not small (e.g., large), we add a "small"style?

Reply:
Its an issue. Will make this.props.size !== RatingSize.Large for small